SWLOR.Game.Server/Feature/AbilityDefinition/Force/ForceInspirationAbilityDefinition.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System.Collections.Generic;
+using SWLOR.Game.Server.Core.NWScript.Enum;
+using SWLOR.Game.Server.Core.NWScript.Enum.VisualEffect;
+using SWLOR.Game.Server.Service.AbilityService;
+using SWLOR.Game.Server.Service.PerkService;
+
+namespace SWLOR.Game.Server.Feature.AbilityDefinition.Force
+{
+    public class ForceInspirationAbilityDefinition: IAbilityListDefinition
+    {
+        public Dictionary<FeatType, AbilityDetail> BuildAbilities()
+        {
+            var builder = new AbilityBuilder();
+            ForceInspiration1(builder);
+            ForceInspiration2(builder);
+            ForceInspiration3(builder);
+
+            return builder.Build();
+        }
+
+        private void Impact(uint activator, uint target, int baseAmount) 
+        {
+            var willpowerMod = GetAbilityModifier(AbilityType.Willpower, activator);
```

Comment:
In an effort to get rid of the modifier stuff, let's change this to raw willpower and dropping it to 15f instead of 30f to compensate.